src/orders/views/OrderDetails/OrderDetailsMessages.tsx

```diff
@@ -366,6 +374,29 @@ export const OrderDetailsMessages: React.FC<OrderDetailsMessages> = ({
         : intl.formatMessage(transactionMessages.success),
     });
   };
+  const handleAddManualTransaction = (
```

Comment:
hmm how about creating another hook eg. useManualTransaction, next to this file, use it here, and test (unit) the hook itself?